sdk/storage/Azure.Storage.Files.DataLake/src/RolePermissionsExtensions.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Text;
+using Azure.Storage.Files.DataLake.Models;
+
+namespace Azure.Storage.Files.DataLake
+{
+    /// <summary>
+    /// Extension methods for RolePermissions.
+    /// </summary>
+    public static class RolePermissionsExtensions
```

Comment:
If we add a static `PathAccessControl` class, we could move all of these methods there and have just one Parse/ToString type.  I'd just tweak the names to be things like `ParseSymbolicRolePermissions` if we move them there.

Reply:
PathAccessControl is already a type in DataLake.Models.  I think another option would be to rename RolePermissionExtensions -> PathAccessControlExtensions, and move the other methods there.